src/Containers/VendorsContainer/VendorsContainer.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import './VendorsContainer.scss';
+import { VendorCard } from '../../Components/VendorCard/VendorCard';
+
+
+export const VendorsContainer = () => {
+  const vendors= useSelector(state => state.vendors);
+    
+  return (
+    <div className='vendor-container'>
+    {vendors.map(vendor => {
```

Comment:
It might be slightly more readable if we take the map out of the return assign it to a variable and then just put the variable within the return with the HTML.